feat(apple): Add info about onCrashedLastRun (#2721)
getsentry/sentry-docs: src/platforms/apple/common/enriching-events/user-feedback.mdx

userFeedback.name = @"John Doe";
[SentrySDK captureUserFeedback:userFeedback];
```

> To capture user feedback of a crash you can use the callback `onCrashedLastRun` of the `SentryOptions`.

Reviewer: ```suggestion
For capturing the User's feedback (only for crashes), you can use the `SentryOptions.onCrashedLastRun` callback.
```

Reviewer: not sure if it's better, but it reads better on my mind.

Author: I would rather stick to the existing version. Maybe @PeloWriter you can help us here a bit with the wording?

> To capture user feedback for a crash you can use the callback `onCrashedLastRun` of the `SentryOptions`.

Reviewer 2: ```suggestion
To capture user feedback regarding a crash, use the callback `onCrashedLastRun` of the `SentryOptions`.
```

Reviewer 2: Let me know if that works better.
